src/Microsoft.DotNet.ProjectModel/FileSystemGlobbing/HashCodeCombiner.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
And then you can remove the dependency from DependencyModel to the HashCodeCombiner.Sources package.